script.js

```diff
@@ -1,66 +1,81 @@
 let currentValue = '';
 let currentOperator = '';
 let result = 0;
-
-function inputNum(value) {
-  currentValue += value;
-  document.getElementById('output').textContent = currentValue;
-}
-
+let displayScreen = document.getElementById('output');
 let operatorButtons = document.getElementsByClassName('operator');
-function inputOperator(oper) {
+document.querySelector('.cal-body').addEventListener('click', function(event){
+  if(event.target.matches('.key')){
+    let buttonValue = event.target.textContent;
+    if(buttonValue==='+'||buttonValue==='‒'||buttonValue==='×'||buttonValue==='÷'||buttonValue==='%'){
+      operatorClicked(buttonValue);
+    }
+    else if(buttonValue === '='){
+      calculate();
+    }
+    else if(buttonValue === 'AC'){
+      resetValues();
+    }
+    else if(buttonValue ==='+/-'){
+      toggleNegate();
+    }
+    else{
+      numericKeyClicked(buttonValue);
+    }
+  }
```

Comment:
```suggestion
   const buttonValue = event.target.textContent;

  switch (buttonValue) {
    case '+':
    case '−':
    case '×':
    case '÷':
    case '%':
      operatorClicked(buttonValue);
      break;
    case '=':
      calculate();
      break;
    case 'AC':
      resetValues();
      break;
    case '+/-':
      toggleNegate();
      break;
    default:
      if (/^[0-9]$/.test(buttonValue)) {
        numericKeyClicked(buttonValue);
      }
  }
```

Reply:
I have made this change but I can't understand what's the betterment in it as there is no repeation and both switch and if statements are serving exactly the same purpose.